--- FIL-plugins-0.3.0/Makefile.orig	2022-01-22 21:42:20.102006017 +0100
+++ FIL-plugins-0.3.0/Makefile	2022-01-22 21:44:43.497895841 +0100
@@ -15,20 +15,22 @@
 #    Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
 
-CPPFLAGS += -I. -fPIC -D_REENTRANT -Wall -O3
+CXX=g++
+CPPFLAGS += -I. -fPIC -D_REENTRANT -Wall
 
+INSTALL_PLUGINS_DIR=/usr/lib/ladspa
 
 all:	filters.so
 
 
 filters.so:	filters.o filters_if.o exp2ap.o
-	g++ -shared filters.o filters_if.o exp2ap.o -o filters.so
+	$(CXX) $(LDFLAGS) $(CXXFLAGS) -shared filters.o filters_if.o exp2ap.o -o filters.so
 
 filters.o:	ladspaplugin.h filters.h
 filters_if.o:	ladspaplugin.h filters.h
 
 install:	all
-	cp  *.so /usr/lib/ladspa
+	cp -p *.so $(DESTDIR)$(INSTALL_PLUGINS_DIR)
 
 clean:
 	/bin/rm -f *~ *.o *.so
